function/transform/transformation.go

```diff
@@ -58,27 +58,17 @@ func ApplyTransform(list api.SeriesList, transform transform, parameters []funct
 	return result, nil
 }
 
-// Derivative estimates the "change per second" between the two samples (scaled consecutive difference)
-func Derivative(values []float64, parameters []function.Value, scale float64) ([]float64, error) {
-	result := make([]float64, len(values))
-	for i := range values {
-		if i == 0 {
-			// The first element has 0
-			result[i] = 0
-			continue
-		}
-		// Otherwise, it's the scaled difference
-		result[i] = (values[i] - values[i-1]) / scale
-	}
-	return result, nil
-}
-
 // Integral integrates a series whose values are "X per millisecond" to estimate "total X so far"
 // if the series represents "X in this sampling interval" instead, then you should use transformCumulative.
 func Integral(values []float64, parameters []function.Value, scale float64) ([]float64, error) {
 	result := make([]float64, len(values))
 	integral := 0.0
 	for i := range values {
+		// Skip the 0th element since thats not technically part of our timerange
```

Comment:
The left most point has no "width" - probably easiest to graph in person. Essentially this makes sure that you can do `transform.derivative | transform.integral` and it'll work properly.